devlib/utils/android.py

```diff
@@ -330,12 +336,18 @@ def _push_pull(self, action, sources, dest, timeout):
         paths = ' '.join(map(do_quote, paths))
 
         command = "{} {}".format(action, paths)
-        if timeout or not self.poll_transfers:
+        if timeout:
             adb_command(self.device, command, timeout=timeout, adb_server=self.adb_server)
         else:
-            with self.transfer_mgr.manage(sources, dest, action):
-                bg_cmd = adb_command_background(self.device, command, adb_server=self.adb_server)
-                self.transfer_mgr.set_transfer_and_wait(bg_cmd)
+            bg_cmd = adb_command_background(self.device, command, adb_server=self.adb_server)
+            handle = PopenTransferHandle(
+                mgr=self.transfer_manager,
```

Comment:
`TypeError(TransferHandleBase.__init__() got an unexpected keyword argument 'mgr')`.
I think this should be `manager`.